examples/cloudflare-worker-loader/alchemy.run.ts

```diff
@@ -0,0 +1,18 @@
+import alchemy from "alchemy";
+import { Worker, WorkerLoader } from "alchemy/cloudflare";
+
+const app = await alchemy("cloudflare-worker-loader");
+
+export const loader = WorkerLoader();
+
+export const worker = await Worker("worker", {
+  entrypoint: "./src/worker.ts",
+  bindings: {
+    LOADER: loader,
+  },
+  url: true,
+});
```

Comment:
can this just be
```
export const worker = await Worker("worker", {
  entrypoint: "./src/worker.ts",
  bindings: {
    LOADER: WorkerLoader(),
  },
  url: true,
});
```

makes the example simpler

Reply:
You also don't need url true.  That's default